alfresco-transformer-base/src/main/java/org/alfresco/transformer/AbstractTransformerController.java

```diff
@@ -352,4 +336,37 @@ private static String messageWithCause(final String prefix, Throwable e)
 
         return sb.toString();
     }
+
+    protected String getTransformerName(File sourceFile, String sourceMimetype, String targetMimetype,
+                                      Map<String, String> transformOptions)
+    {
+        long sourceSizeInBytes = sourceFile.length();
+        String transformerName = transformRegistry.getTransformerName(sourceMimetype, sourceSizeInBytes,
+                targetMimetype, transformOptions, null);
+        if (transformerName == null)
+        {
+            throw new TransformException(BAD_REQUEST.value(), "No transforms were able to handle the request");
+        }
+        return transformerName;
+    }
+
+    protected Map<String, String> createTransformOptions(Object... namesAndValues)
```

Comment:
Replace `Object...` with `Map<String, Object>`.

Reply:
That would be nice, but also needs to not include keys to null values.
I need to check again, but it appears that alfresco-transform-core is Java 8 rather than >= 9, so can't use Map.of either and then remove the nulls later.

Reply:
Should we agree to switch to minimum Java Source = 11 ? 

Also aligns with ACS Repo (https://github.com/Alfresco/alfresco-repository/blob/master/pom.xml#L38) and AIS.

Reply:
I understand the need for this method now. Please disregard this comment/conversation.